### PR TITLE
Translation of Git episode 4

### LINKED ISSUE #64: Translation: git-novice Episode 4
Translation: git-novice Section 4 "Tracking Changes"
-----

* Please assign yourself (or add a comment) to the issue **before** you start translating, so other people know that you are translating.
* Once you have finished translating, send a pull request and reference this issue.

翻訳：git-novice セクション4 "Tracking Changes"
-----

* あなたが翻訳し始めたと分かるように、翻訳を**始める前**にこのイシューを自分に割り振って(assign)、あるいはコメントを書いて下さい。
* 翻訳が終わったら、プルリクエストを開いて、このイシューを参照・リンクして下さい。

https://github.com/swcarpentry-ja/i18n/blob/b3225373bbf9b2f14b8882e4bc019cbd9dbad114/po/git-novice.ja.po#L2058

### PR DESCRIPTION
**Pull request by rikutakei**
Turns out this lesson was **way** longer than I thought it was, so it took a while for me to get it done.

よろしくお願いします！

### CODE REVIEW

File: po/git-novice.ja.po [hunk 1]
```diff
@@ -2010,35 +2010,35 @@ msgid ""
...
+"title: 変更内容の記録\n"
 "teaching: 20\n"
 "exercises: 0\n"
 "questions:\n"
-"- \"How do I record changes in Git?\"\n"
-"- \"How do I check the status of my version control repository?\"\n"
-"- \"How do I record notes about what changes I made and why?\"\n"
+"- \"どうやって変更点を Git で記録するの？\"\n"
+"- \"どうやってリポジトリの状態をチェックするの？\"\n"
+"- \"何を変更して、なぜ変えたのかをメモするにはそうすればいいの？\"\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"- \"何を変更して、なぜ変えたのかをメモするにはどうすればよいですか？\"\n"
```

File: po/git-novice.ja.po [hunk 2]
```diff
@@ -2010,35 +2010,35 @@ msgid ""
...
 "---\n"
-"title: Tracking Changes\n"
+"title: 変更内容の記録\n"
 "teaching: 20\n"
 "exercises: 0\n"
 "questions:\n"
-"- \"How do I record changes in Git?\"\n"
-"- \"How do I check the status of my version control repository?\"\n"
-"- \"How do I record notes about what changes I made and why?\"\n"
+"- \"どうやって変更点を Git で記録するの？\"\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: I think we should follow the style set by @masamiy [for R](https://swcarpentry-ja.github.io/r-novice-gapminder/ja/index/index.html)
```suggestion
"- \"どうやって変更点を Git で記録出来ますか？\"\n"
```

File: po/git-novice.ja.po [hunk 3]
```diff
@@ -2010,35 +2010,35 @@ msgid ""
...
+"- \"どうやってリポジトリの状態をチェックするの？\"\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"- \"どうやってリポジトリの状態をチェック出来ますか？\"\n"
```

File: po/git-novice.ja.po [hunk 4]
```diff
@@ -2010,35 +2010,35 @@ msgid ""
...
+"- \"何を変更して、なぜ変えたのかをメモするにはそうすればいいの？\"\n"
 "objectives:\n"
-"- \"Go through the modify-add-commit cycle for one or more files.\"\n"
-"- \"Explain where information is stored at each stage of that cycle.\"\n"
-"- \"Distinguish between descriptive and non-descriptive commit messages.\"\n"
+"- \"「変更・追加・コミット」の作業を一つ以上のファイルで使う。\"\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: 「作業を使う」sounds odd to me
Also, applying @masamiy 's style (e.g., [R episode 1](https://swcarpentry-ja.github.io/r-novice-gapminder/ja/_episodes/01-rstudio-intro/index.html)): for goals, use 「しましょう」
```suggestion
"- \"一つ以上のファイルにおいて、「変更・追加・コミット」の作業を行いましょう。\"\n"
```

File: po/git-novice.ja.po [hunk 5]
```diff
@@ -2010,35 +2010,35 @@ msgid ""
...
+"- \"「変更・追加・コミット」を行っている際、情報がどこに保管されているのか説明する。\"\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"- \"「変更・追加・コミット」を行っている際、情報がどこに保管されているのか説明しましょう。\"\n"
```

File: po/git-novice.ja.po [hunk 6]
```diff
@@ -2010,35 +2010,35 @@ msgid ""
...
+"- \"わかりやすいコミットメッセージと、そうでないメッセージの違いを区別する。\"\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"- \"わかりやすいコミットメッセージと、そうでないメッセージの違いを区別できるようになりましょう。\"\n"
```

File: po/git-novice.ja.po [hunk 7]
```diff
@@ -2080,17 +2080,17 @@ msgid ""
...
-"We'll use `nano` to edit the file;\n"
-"you can use whatever editor you like.\n"
-"In particular, this does not have to be the `core.editor` you set globally "
-"earlier. \n"
-"But remember, the.language-bash command to create or edit a new file will "
-"depend on the editor you choose (it might not be `nano`). \n"
-"For a refresher on text editors, check out [\"Which Editor?\"](https://"
-"swcarpentry.github.io/shell-novice/03-create/) in [The Unix Shell](https://"
-"swcarpentry.github.io/shell-novice/) lesson."
+"赤い惑星（火星）が基地に最適かどうかについてのノートが書いてある\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: This is a problem with the original, but I think it's even more apparent when translated... it is confusing to say that the file "contains some notes" when it hasn't been created yet!
```suggestion
"赤い惑星（火星）が基地に最適かどうかについてのノートを書くために\n"
```

File: po/git-novice.ja.po [hunk 8]
```diff
@@ -2192,9 +2192,9 @@ msgid ""
 "that Git isn't keeping track of.\n"
 "We can tell Git to track a file using `git add`:"
 msgstr ""
-"The \"untracked files\" message means that there's a file in the directory\n"
-"that Git isn't keeping track of.\n"
-"We can tell Git to track a file using `git add`:"
+"\"untracked files\" （追跡されていないファイル）というメッセージは、\n"
+"Git が追跡していないファイルがこのディレクトリに存在していることを意味しています。\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"Git が追跡していないファイルがこのディレクトリに存在していることを意味します。\n"
```

File: po/git-novice.ja.po [hunk 9]
```diff
@@ -2245,10 +2245,10 @@ msgid ""
 "To get it to do that,\n"
 "we need to run one more command:"
 msgstr ""
-"Git now knows that it's supposed to keep track of `mars.txt`,\n"
-"but it hasn't recorded these changes as a commit yet.\n"
-"To get it to do that,\n"
-"we need to run one more command:"
+"これで Git は `mars.txt` を追跡することができます。\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: A little more complicated wording, but I think this is closer to the English (not sure which is better...)
```suggestion
"これで Git は `mars.txt` を追跡するように設定することができました。\n"
```

File: po/git-novice.ja.po [hunk 10]
```diff
@@ -2322,17 +2322,17 @@ msgid ""
...
-"sentence \"If applied, this commit will\" <commit message here>.\n"
-"If you want to go into more detail, add a blank line between the summary "
-"line and your additional notes. Use this additional space to explain why you "
-"made changes and/or what their impact will be."
+"[良いコミットメッセージ][commit-messages] は、どのような変更が"
+"行われたのかが分かる短い文（50字以下）\n"
+"で始まります。大抵のメッセージは、「このコミットを適用すると<コミットメッセージ>」"
+"という文が完全文になるように書かれます。\n"
+"もっと詳しく書きたい場合は、このメッセージの後に改行してから"
+"ノートを加えましょう。この追加されたスペースには、なぜこのような変更を"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: I think literally using スペース might be confusing
```suggestion
"ノートを加えましょう。ここには、なぜこのような変更を"
```

File: po/git-novice.ja.po [hunk 11]
```diff
@@ -2579,19 +2579,19 @@ msgid ""
...
 
 # ordered list
 #: git-novice/_episodes/04-changes.md:278
 msgid ""
 "1.  The first line tells us that Git is producing output similar to the Unix "
 "`diff` command"
 msgstr ""
-"1.  The first line tells us that Git is producing output similar to the Unix "
-"`diff` command"
+"1.  最初の文は、Git が 古いバージョンと新しいバージョンのファイルをUnix の "
```
Comment (joelnitta) [inline po/git-novice.ja.po]: extra space
```suggestion
"1.  最初の文は、Git が古いバージョンと新しいバージョンのファイルをUnix の "
```

File: po/git-novice.ja.po [hunk 12]
```diff
@@ -2608,22 +2608,22 @@ msgid ""
...
-"    `df0654a` and `315bf3a` are unique computer-generated labels for those "
-"versions.\n"
-"3.  The third and fourth lines once again show the name of the file being "
-"changed.\n"
-"4.  The remaining lines are the most interesting, they show us the actual "
-"differences\n"
-"    and the lines on which they occur.\n"
-"    In particular,\n"
-"    the `+` marker in the first column shows where we added a line."
+"    コマンドと同じように、二つのファイル比べていることを表しています。\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"    コマンドと同じように、二つのファイルを比べていることを表しています。\n"
```

File: po/git-novice.ja.po [hunk 13]
```diff
@@ -2608,22 +2608,22 @@ msgid ""
...
+"    表示しています。\n"
+"    `df0654a` と `315bf3a` はコンピュータが作成した、各バージョン"
+"につけられたIDです。\n"
+"3.  3・4つ目の文章は、変更されているファイルの名前を"
+"表示しています。\n"
+"4.  残りの文章が一番重要です。ここに、"
+"何が変わり、\n"
+"    どの行が変更されたのかが記されています。\n"
+"    特に、この\n"
+"    `+` マークは、どこに文章を加えたのかを表しています。"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: keep in line with the previous 受け身. Also, so as not to confuse with what **git** is doing.
```suggestion
"    `+` マークは、どこに文章が加えられたのかを表しています。"
```

File: po/git-novice.ja.po [hunk 14]
```diff
@@ -2739,23 +2739,23 @@ msgid ""
...
+"> `git commit` と入力した際にステージングエリアに何もなかった場合、\n"
+"> Git は`git commit -a` もしくは `git commit --all` を入力するように言ってきます。\n"
+"> このコマンドは、写真を撮る時の*「全員集合！」* のようなもので、全ての変更点を強制的にコミットできます。\n"
+"> ですが、大抵は、コミットしたい変更点のみを\n"
+"> ステージングエリアに入れるほうが良いでしょう。これによって、\n"
+"> 不要な変更点を間違えてコミットすることもありません。（集合写真の例に例えると、\n"
+"> メイクの不完全なエキストラがステージの上を横断しているのを\n"
+"> 一緒に撮ってしまった、みたいなものです。）\n"
+"> ですので、ステージングエリアにどの変更点を入れるかは自分で管理しましょう。\n"
+"> さもないと、必要以上に\"git コミット やり直す\"で\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po:2757]: Not a literal translation, but 何となくこの方がピンと来るかな？
```suggestion
"> さもないと、必要以上に\"git コミット やり直す\"を\n"
```

File: po/git-novice.ja.po [hunk 15]
```diff
@@ -2739,23 +2739,23 @@ msgid ""
...
+"> 検索をかけることになるでしょう。"
```
Comment (joelnitta) [inline po/git-novice.ja.po:2758]:
```suggestion
"> ググることになるでしょう。"
```

File: po/git-novice.ja.po [hunk 16]
```diff
@@ -2769,11 +2769,11 @@ msgid ""
...
 "we'll add another line to the file:"
 msgstr ""
-"Let's watch as our changes to a file move from our editor\n"
-"to the staging area\n"
-"and into long-term storage.\n"
-"First,\n"
-"we'll add another line to the file:"
+"それでは、ファイルの変更点がエディタから\n"
+"ステージングエリア、そして\n"
+"長期保存される工程を見てみましょう。\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"長期保存に移る過程を見てみましょう。\n"
```

File: po/git-novice.ja.po [hunk 17]
```diff
@@ -2973,15 +2973,15 @@ msgid ""
...
+"> 時折、例えばテキストドキュメントなど、列ごとの違いを表示するのは\n"
+"> 大雑把すぎる場合があります。こういう時は、`git diff` の `--color-words` オプションを\n"
+"> 使えば、色で文字ごとの違いを表示してくれるので、\n"
+"> 大変便利です。"
 
 # blockquote, which can be cascaded
 #: git-novice/_episodes/04-changes.md:501
 msgid "> ## Paging the Log"
-msgstr "> ## Paging the Log"
+msgstr "> ## ログをページ化する"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: I think ページ化する may be a bit technical
```suggestion
msgstr "> ## ログをめくる"
```

File: po/git-novice.ja.po [hunk 18]
```diff
@@ -3120,23 +3120,23 @@ msgid ""
 "> ~~~"
 msgstr ""
 ">\n"
-"> You can also combine the `--oneline` options with others. One useful\n"
-"> combination is:\n"
+"> `--oneline` オプションを他のオプションと組み合わせることもできます。便利なコンビネーションの\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"> `--oneline` オプションを他のオプションと組み合わせることもできます。便利な組み合わせの\n"
```

File: po/git-novice.ja.po [hunk 19]
```diff
@@ -3217,10 +3217,10 @@ msgid ""
 "(`git add`) and then commit the staged changes to the\n"
 "repository (`git commit`):"
 msgstr ""
-"To recap, when we want to add changes to our repository,\n"
-"we first need to add the changed files to the staging area\n"
-"(`git add`) and then commit the staged changes to the\n"
-"repository (`git commit`):"
+"まとめると、変更内容をリポジトリに追加したい時は\n"
+"`git add` で変更点をステージングエリアに入れてから、\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: sticking with the "moving things through different places" paradigm
```suggestion
"`git add` で変更点をステージングエリアに移してから、\n"
```

File: po/git-novice.ja.po [hunk 20]
```diff
@@ -3308,18 +3308,18 @@ msgstr ""
...
+"> > ## 回答\n"
 "> >\n"
-"> > 1. Would only create a commit if files have already been staged.\n"
-"> > 2. Would try to create a new repository.\n"
-"> > 3. Is correct: first add the file to the staging area, then commit.\n"
-"> > 4. Would try to commit a file \"my recent changes\" with the message "
-"myfile.txt."
+"> > 1. ファイルがステージングエリアにない限り、コミットできません。\n"
+"> > 2. 新しくリポジトリを作ろうとします。\n"
+"> > 3. 正しい回答です。まずファイルをステージングエリアに追加し、それからコミットしています。\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"> > 3. 正しい回答です。まずファイルをステージングエリアに追加し、それからコミットします。\n"
```

File: po/git-novice.ja.po [hunk 21]
```diff
@@ -3619,40 +3619,40 @@ msgid ""
 "> > {: .output}"
 msgstr ""
 ">\n"
-"> For each of the commits you have done, Git stored your name twice.\n"
-"> You are named as the author and as the committer. You can observe\n"
-"> that by telling Git to show you more information about your last\n"
-"> commits:\n"
+"> 今までのコミットには、Git はあなたの名前を二回保存しています。\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: This is in the past tense; also should specify that it's for each commit
```suggestion
"> 今までのコミットの一つにつき、Git はあなたの名前を二回保存しました。\n"
```

**Review by joelnitta — COMMENTED** [review]
Thanks for the PR! Please see my comments and suggestions.

File: po/git-novice.ja.po (hunk 15 above)
Comment (rikutakei) [reply po/git-novice.ja.po:2758]: ここは「ググる」よりも「検索」のほうが丁寧かと思います。他の場所で丁寧語（です・ます・しましょう）を使っているから、ここもそうしたほうが良いんじゃないかな?

**Review by joelnitta — APPROVED** [review]

**Pull request merged**